scripts/validate-nimbus-file.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+
+TARGET_PATH=/tmp/nimbus-cli
```

Comment:
Why not use `./dist/scripts`? We use this now for TypeScript cronjobs.

Reply:
`./dist/scripts` works just as well to install `nimbus-cli` to — [15a8985](https://github.com/mozilla/blurts-server/pull/4861/commits/15a8985aa42a59fa939fb7cd628cf177188cb746).

Reply:
@rhelmer Thinking about it again I’ve reverted the change in [dcc316b](https://github.com/mozilla/blurts-server/commit/dcc316be86feb7bb2a3981e8a1b382b3fd88aa21). I think my preference would be to install `nimbus-cli` to `/tmp`. We only need it to verify `nimbus.yaml` at build time. If we install it to `./dist/scripts` we would make it unnecessarily part of the code we deploy?